julia/README-DEV.md

```diff
@@ -0,0 +1,13 @@
+# Workflow for making a release
```

Comment:
Please add appropriate instructions to https://cwiki.apache.org/confluence/display/MXNET/Release+Process

Reply:
Sell, since I do not experience with making a release with `git subtree`, not sure what kind of problem will pop up. But I can write something down.

Reply:
Feel free to work together with @cjolivier01 and @anirudh2290 , they will be managing the next release. Although I don't think this will be part of 1.2, they should be of help. Alternatively, a few old release managers: @eric-haibin-lin @mbaijal

Reply:
Agree that such instructions should be added to cwiki